packages/taiko-client/proposer/proposer.go

```diff
@@ -159,11 +163,49 @@ func (p *Proposer) InitFromConfig(ctx context.Context, cfg *Config) (err error)
 
 // Start starts the proposer's main loop.
 func (p *Proposer) Start() error {
-	p.wg.Add(1)
-	go p.eventLoop()
+	startRPCServer(p)
+
+	// p.wg.Add(1)
```

Comment:
@smartprogrammer93 Here I disable default proposing, we may add some env variable for this, or just delete this code.